.editorconfig

```diff
@@ -0,0 +1,14 @@
+# editorconfig.org
+
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
@paulrobertlloyd 👀 never knew this was a thing.

Reply:
Couldn’t live without it! It’s like an oldskool attempt at what Prettier tends to be used for today.

Reply:
I think nearly every project I've worked on seems to have settled on that style... Not come across a "tabs instead of spaces" project for ages... (phew)